### PR TITLE
Add documentation and environment file for flannel

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -230,10 +230,35 @@ single master node you can disable loadbalancer node by including
...
 ```
 
+== Select SDN Type ==
+
+By default, OpenShift is deployed with https://docs.openshift.com/enterprise/3.2/architecture/additional_concepts/sdn.html[OpenShift-SDN].
+When used with Openstack Neutron with GRE or VXLAN tunnels, packets are encapsulated twice
+which can have an impact on performances. Those Heat templates allow using https://github.com/coreos/flannel[Flannel]
+instead of openshift-sdn, with the `host-gw` backend to avoid the double encapsulation.
+To do so, you need to include the `env_flannel.yaml` environment file when you create the stack:
+
```
Comment (jprovaznik) [inline README.adoc]: shouldn't it be "openshift-on-openstack/env_flannel.yaml" (according to the fact that newely added env file is named env_flannel.yaml?

File: README.adoc [hunk 2]
```diff
@@ -230,10 +230,35 @@ single master node you can disable loadbalancer node by including
...
+```bash
+heat stack-create my_openshift \
+   -e openshift_parameters.yaml \
+   -f openshift-on-openstack/openshift.yaml \
+   -e env_sdn_flannel.yaml
```
Comment (jprovaznik) [inline README.adoc]: shouldn't it be "openshift-on-openstack/env_flannel.yaml" (according to the fact that newely added env file is named env_flannel.yaml?

File: README.adoc [hunk 3]
```diff
@@ -230,10 +230,35 @@ single master node you can disable loadbalancer node by including
...
+```
+
+To use this feature, the Neutron `port_security` extension driver needs to enabled.
```
Comment (jprovaznik) [inline README.adoc]: /to enabled/to be enabled/?

**Comment by jprovaznik** [comment]
Thanks! Only twi nits commented, otwerwise LGTM

File: README.adoc (hunk 2 above)
Comment (lebauce) [inline README.adoc]: Good catch. Thanks

File: README.adoc (hunk 3 above)
Comment (lebauce) [inline README.adoc]: Fixed

**Pull request merged**